.github/renovate.json

```diff
@@ -89,6 +91,7 @@
     },
     {
       "extends": ["schedule:weekly"],
+      "baseBranches": ["v1.29.x"],
```

Comment:
I think `baseBranches` is top level option: https://docs.renovatebot.com/configuration-options/#basebranches